### DISCUSSION TITLE
Option for launching Terminal minimized to tray on startup

### DISCUSSION THREAD

**Discussion by alchem1no** [microsoft/terminal#18011] [Ideas]
Hi! I enjoy using this terminal for PowerShell and WSL distros. Also it has two great options which I really like - launching on startup and dropdown mode (called Quake window in the Terminal). I would like Terminal to launch minimized to tray and in Quake window so when I boot my PC and log into it I could access Terminal right away with `` Win + ` `` shortcut.
However, Terminal can only launch maximized on startup and I haven't figured out how to launch it in Quake window rather than usual window. So when I log into the PC instead of prefered behavior i get Terminal launched maximized and I have to additionally press **Win + `** to activate dropdown menu. That are features I really miss in this program.

Also there is an option to hide Terminal in notification are when user minimizes Terminal window but no option for doing that when user closes the window. I want Terminal to hide to tray but I also want it to be accessible via `Alt + Tab`.

I suggest adding these features:
1. Option to launch Terminal on system startup minimized to tray (just like in Phone Link app);
2. Option for hiding Terminal in tray on window close (i like how it is implemented in Netlimiter);
3. Option for launching Terminal in Quake window on startup.

**Comment by v1ckxy** [comment]
Let's up this.

**Comment by leandrumartin** [comment]
Agree. I only turn on the "open on system startup" option so that I can keep the app in the tray to access Quake Mode at any time—but I don't actually ever want to *use* Terminal at system startup, just to have it start running in the background at startup.

**Comment by edvardeishen** [comment]
Still not present🤣. As always, Microsoft adds feature, doesn't finish it and then fully abandons

**Reply by zadjii-msft** [reply]
The terminal is a community project, and you are more than welcome to submit a pull request for any features you would like to contribute 😉 

As it stands, I haven't been active on the Terminal project much since I wrote up https://github.com/microsoft/terminal/issues/9996#issuecomment-1561249825, but IIRC that thread has my notes from when I was more active here.
